__mocks__/got.js

```diff
@@ -0,0 +1,73 @@
+const people = require('./../assets/people.json');
+const films = require('./../assets/films.json');
+const species = require('./../assets/species.json');
+const planets = require('./../assets/planets.json');
+
+
+const got = (url, options) => {
```

Comment:
Why isn't this async? The real `got` package is async/await, but your mock isn't